pyvista/plotting/plotting.py

```diff
@@ -5798,6 +5803,10 @@ def on_timer(iren, event_id):
         self._initialized = True
         log.debug('Plotter init stop')
 
+    @property
+    def render_window(self):
+        return self.ren_win
```

Comment:
this probably needs some safety catches, testing, and documentation.

We use `ren_win` and `hassattr(self, 'ren_win') all over the place and should try to clean all of that up into this fully name propery